### DISCUSSION TITLE
Write to file fonts list

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [ImageMagick/ImageMagick#2325] [Help] [answered]
convert -list font -write fonts.txt

Simple command, but didn't work.

Any suggestion?

**Comment by dlemstra** [comment, marked as answer]
The output of `-list` is written to the output and you cannot use `-write` to write it to a file. You could do the following instead: `convert -list font > fonts.txt`.

**Reply by jaka1-newbie** [reply]
Great! Thx!
